docs/installation-configuration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: VRA Installation & Configuration
+meta_desc: Information on how to install the VRA provider.
+layout: installation
+---
+
+## Installation
+
+The Pulumi VRA provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumiverse/vra`](https://www.npmjs.com/package/@pulumiverse/vra)
+* Python: [`pulumiverse_vra`](https://pypi.org/project/pulumiverse-vra/)
+* Go: [`github.com/pulumiverse/pulumi-vra/sdk/go/vra`](https://pkg.go.dev/github.com/pulumiverse/pulumi-vra/sdk)
+* .NET: [`Pulumiverse.VRA`](https://www.nuget.org/packages/Pulumiverse.VRA)
+
```

Comment:
Can you add a section here explaining how to manually install the plugin binary?

Here is the snippet from the Talos provider as an example:

https://github.com/pulumiverse/pulumi-talos/blob/f64635341c178019cd59f9a72819d56d5c29f6b0/docs/installation-configuration.md#L16-L23